packages/next/src/withTRPC.tsx

```diff
@@ -18,8 +18,13 @@ import {
   NextPageContext,
 } from 'next/dist/shared/lib/utils';
 import React, { createElement, useState } from 'react';
-import { QueryClient, QueryClientProvider } from 'react-query';
-import { DehydratedState, Hydrate, dehydrate } from 'react-query/hydration';
+import {
+  DehydratedState,
+  Hydrate,
```

Comment:
Do you know at which minor this was changed in react-query 3? We would need to update `peerDependencies` to that version as well in

Reply:
The change was made in https://github.com/tannerlinsley/react-query/releases/tag/v3.22.0. I double checked the `peerDependencies`, all of them are above that.

Reply:
Perfect